commit 40a63c2461142314f475a7498badf6596f950c2b (refs/changes/70/3870/2)
Author: Jason King <jason.king@joyent.com>
Date:   2018-04-25T19:48:54-05:00 (1 year, 5 months ago)
    
    OS-6908 Makefiles missing 'all' target
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/usr/src/cmd/varpd/Makefile b/usr/src/cmd/varpd/Makefile
index 215f23d29e..8c29829d40 100644
--- a/usr/src/cmd/varpd/Makefile
+++ b/usr/src/cmd/varpd/Makefile
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2015 Joyent, Inc.
+# Copyright 2018 Joyent, Inc.
 #
 
 PROG= varpd
@@ -43,6 +43,8 @@ C99LMODE=	-Xc99=%all
 
 .KEEP_STATE:
 
+all: $(PROG)
+
 $(PROG): $(OBJS)
 	$(LINK.c) -o $@ $(OBJS) $(LDLIBS)
 	$(POST_PROCESS)
diff --git a/usr/src/cmd/vndadm/Makefile b/usr/src/cmd/vndadm/Makefile
index aa9c22d296..2b9ca6c3c1 100644
--- a/usr/src/cmd/vndadm/Makefile
+++ b/usr/src/cmd/vndadm/Makefile
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright (c) 2014 Joyent, Inc.  All rights reserved.
+# Copyright (c) 2018 Joyent, Inc.  All rights reserved.
 #
 
 PROG= vndadm
@@ -38,6 +38,8 @@ SUBDIRS = test
 
 .KEEP_STATE:
 
+all: $(PROG)
+
 $(PROG): $(OBJS)
 	$(LINK.c) -o $@ $(OBJS) $(LDLIBS)
 	$(POST_PROCESS)
